proposals/10-egress-gateways.md

```diff
@@ -36,6 +36,21 @@ ingress use case.
   cloud services to workloads, I need fail-over from one cloud provider to
   others when the primary cloud provider is overwhelmed or in a failure state.
 
+* As a gateway admin providing egress routing to external services, I need
```

Comment:
I almost feels like this conflates two types of identity that's worth pulling out. When verifying the server, that's TLS. When we present the client identity is that necessarily TLS? JWT? Basic auth?

Reply:
Push didn't complete, e99ee0675493e5b1742b37e4fddc8943383a6a71